meteor/lib/typings/webmanifest.d.ts

```diff
@@ -0,0 +1,206 @@
+	 */
+	protocol_handlers?: ProtocolHandler[]
+	[k: string]: unknown
+}
+export interface ManifestImageResource {
+	/**
+	 * The sizes member is a string consisting of an unordered set of unique space-separated tokens which are ASCII case-insensitive that represents the dimensions of an image for visual media.
+	 */
+	sizes?: string | 'any'
+	/**
+	 * The src member of an image is a URL from which a user agent can fetch the icon's data.
+	 */
+	src: string
+	/**
+	 * The type member of an image is a hint as to the media type of the image.
+	 */
+	type?: string
+	purpose?:
+		| 'monochrome'
+		| 'maskable'
+		| 'any'
+		| 'monochrome maskable'
+		| 'monochrome any'
+		| 'maskable monochrome'
+		| 'maskable any'
+		| 'any monochrome'
+		| 'any maskable'
+		| 'monochrome maskable any'
+		| 'monochrome any maskable'
+		| 'maskable monochrome any'
+		| 'maskable any monochrome'
+		| 'any monochrome maskable'
+		| 'any maskable monochrome'
+	[k: string]: unknown
+}
+export interface ExternalApplicationResource {
+	/**
+	 * The platform it is associated to.
+	 */
+	platform: 'chrome_web_store' | 'play' | 'itunes' | 'windows'
+	/**
+	 * The URL where the application can be found.
+	 */
+	url?: string
+	/**
+	 * Information additional to the URL or instead of the URL, depending on the platform.
+	 */
+	id?: string
+	/**
+	 * Information about the minimum version of an application related to this web app.
+	 */
+	min_version?: string
+	/**
+	 * An array of fingerprint objects used for verifying the application.
+	 */
+	fingerprints?: {
+		type?: string
+		value?: string
+		[k: string]: unknown
+	}[]
+	[k: string]: unknown
+}
+/**
+ * A shortcut item represents a link to a key task or page within a web app. A user agent can use these values to assemble a context menu to be displayed by the operating system when a user engages with the web app's icon.
+ */
+export interface ShortcutItem {
+	/**
+	 * A string that represents the id of the shortcut item
+	 */
+	id?: string
+	/**
+	 * The name member of a shortcut item is a string that represents the name of the shortcut as it is usually displayed to the user in a context menu.
+	 */
+	name: string
+	/**
+	 * The short_name member of a shortcut item is a string that represents a short version of the name of the shortcut. It is intended to be used where there is insufficient space to display the full name of the shortcut.
+	 */
+	short_name?: string
+	/**
+	 * The description member of a shortcut item is a string that allows the developer to describe the purpose of the shortcut.
+	 */
+	description?: string
+	/**
+	 * The url member of a shortcut item is a URL within scope of a processed manifest that opens when the associated shortcut is activated.
+	 */
+	url: string
+	/**
+	 * The icons member of a shortcut item serves as iconic representations of the shortcut in various contexts.
+	 */
+	icons?: ManifestImageResource[]
+	[k: string]: unknown
+}
+
+export interface ProtocolHandler {
+	protocol:
```

Comment:
I'm going to assume we're not gonna be using Sofie UI for bitcoin transactions? :)

Reply:
Sofie NFT: we're going to automatically mint only 19050607 individual seconds of newscasts, so you can own a slice of reality! And an NFT is forever, so it's potentially worth billions!

Reply:
In all seriousness, ProtocolHandler interface is just a direct translation of this standard document: https://wicg.github.io/manifest-incubations/#protocol_handlers-member and the related definition of white-listed protocols here: https://html.spec.whatwg.org/multipage/system-state.html#normalize-protocol-handler-parameters, with the addition of the new scheme of `web+some_arbitrary_token`.